src/lsp-server.ts

```diff
@@ -174,7 +174,7 @@ export class LspServer {
         if (typescriptVersion) {
             this.logger.info(`Using Typescript version (${typescriptVersion.source}) ${typescriptVersion.versionString} from path "${typescriptVersion.tsServerPath}"`);
         } else {
-            throw Error('Could not find a valid tsserver version. Exiting.');
+            throw Error('Could not find a valid tsserver version on $PATH. Please install tsserver with `npm install -g typescript`. Exiting.');
```

Comment:
`findTypescriptVersion` will try to find the typescript dependency either in the workspace or globally so assuming that typescript was not found in the PATH and suggesting to install it globally is only half-correct.

Also some users might be using `yarn` or other package manager so maybe better leave out the exact command.

I suppose the message could mention both possibilities like:

```suggestion
            throw Error('Could not find a valid tsserver version in the workspace or in the $PATH. Please ensure that the "typescript" dependency is installed in either location. Exiting.');
```

Reply:
Ah, that's a good point. What about

> Could not find a valid `tsserver` executable in the workspace or in the $PATH. Please ensure that the "typescript" dependency is installed in either location. Exiting.

That way it is more clear about what binary executable needs to be available.

Reply:
Sounds good.

Reply:
Are you planning to continue here or you want me to take over?

Reply:
Ah, I didn't realize this was outstanding on my part. Updated to the proposed wording in 7f662ba92ddf5f40935e02f24310e5e430802e25